TeensySF/config.ini

```diff
@@ -1 +1,20 @@
-humstart=1890
+# TeensySF config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+volHum=15
```

Comment:
How about we just comment out entries which have the default values?
Like: # volHum=15

Reply:
I think that's a drastic change considering the millions of config.ini's already existing in the world which are formatted this way already. 
Are you sure that's how we should do this for all of these?